File: 102-2_iOS程式設計/SliderDemo/SliderDemo/ViewController.m
//
//  ViewController.m
//  SliderDemo
//
//  Created by CSIEVNUT403 on 2014/3/13.
//  Copyright (c) 2014年 CSIEVNUT403. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (IBAction)onSliderValueChanged:(id)sender
{
    UISlider *slider = (UISlider *) sender;
    NSString *sliderValue = [[NSString alloc] initWithFormat:@"%d", (int) [slider value]];

    [self.label setText:sliderValue];
}

- (void)viewWillAppear:(BOOL)animated
{
    [self.label setText:@"5"];
}

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 105-2_iOS程式設計/MulitButton/MulitButton/ViewController.swift
//
//  ViewController.swift
//  MulitButton
//
//  Created by mojado on 2017/3/8.
//  Copyright © 2017年 VNU. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var labelTel: UILabel!
    @IBOutlet weak var labelMsg: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    @IBAction func sureClick(sender: UIButton) {
        if labelTel.text?.lengthOfBytesUsingEncoding(NSUTF8StringEncoding) == 10 {
            labelMsg.text = "撥打電話：" + labelTel.text!
        } else {
            labelMsg.text = "必須輸入１０個數字！"
        }
    }
    
    @IBAction func clearClick(sender: UIButton) {
        labelTel.text = ""
    }
    
    @IBAction func numberClick(sender: UIButton) {
        labelTel.text = labelTel.text! + sender.currentTitle!
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

